packages/adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+const List<TargetPlatform> _desktop = <TargetPlatform>[
+  TargetPlatform.fuchsia,
+  TargetPlatform.linux,
+  TargetPlatform.macOS,
+  TargetPlatform.windows
+];
+const List<TargetPlatform> _mobile = <TargetPlatform>[
+  TargetPlatform.iOS,
+  TargetPlatform.android
+];
+
+/// A group of standard breakpoints built according to the material
+/// specifications for screen width size.
+class Breakpoints {
+  /// This is a standard breakpoint that can be used as a fallthrough in the
+  /// case that no other breakpoint is active.
+  ///
+  /// It is active from a width of -1 dp to infinity.
+  static const Breakpoint standard = WidthPlatformBreakpoint(begin: -1);
+
+  /// This is the defined small breakpoint. Commonly used to indicate mobile but
```

Comment:
This kind of documentation is pretty close https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation. Here and below.

It's definitely important to point out that this breakpoint typically indicates a mobile screen. 

The API doc could just be: A mobile screen or window whose width is less than 600 dp.